books/system/doc/acl2-doc.lisp

```diff
@@ -109677,59 +109573,86 @@ arithmetic) for libraries of @(see books) for arithmetic reasoning.</p>")
  @('observation-cw') during @(tsee include-book), then you need to manage that
  yourself.</p>")
 
-(defxdoc obtaining-acl2
-  :parents (installation-support)
-  :short "Alternate instructions for obtaining the ACL2 sources and community books"
-  :long "<p>This topic may be avoided by reading the installation summary: see
-  @(see installation-summary), which is intended to be self-contained.</p>
+(defxdoc obtaining-common-lisp
+  :parents (installation)
+  :short "Obtaining Common Lisp"
+  :long "<p>A requirement for installing ACL2 is to install a suitable
+ Common Lisp implementation.  ACL2 can be hosted by several Common Lisp
+ implementations, as listed alphabetically below.  The most commonly-used of
+ these are SBCL and CCL, which can be obtained without charge.</p>
```

Comment:
Perhaps it should be mentioned here that 64-bit versions should be preferred over 32-bit versions of any of the following Common Lisps (for performance and to certify certain community books; see the Zulip discussion). This is mentioned in `:doc ccl-installation`, but at a glance I don't see it mentioned explicitly anywhere else.

Reply:
Good idea.  I haven't gone over that page yet but will try to include something like that when I do.